metacatalog/util/exceptions.py

```diff
@@ -8,3 +8,7 @@ class MetadataMissingError(RuntimeError):
 
 class IOOperationNotFoundError(RuntimeError, ValueError):
     pass
+
+
+class StandardsExportExtentionNotFoundError(RuntimeError, ValueError):
+    pass
```

Comment:
What do we use this exception for?

Reply:
That's a good question, I used this Exception before, but not anymore.  
I will remove it.